fdbserver/BlobManifest.actor.cpp

```diff
@@ -614,10 +615,11 @@ class BlobManifestDumper : public ReferenceCounted<BlobManifestDumper> {
 		loop {
 			state std::vector<BlobManifestFile> allFiles = wait(BlobManifestFile::listAll(writer));
 			int count = BlobManifest::count(allFiles);
+			TraceEvent("BlobManfiestCleanup").detail("Count", count);
 			if (count <= SERVER_KNOBS->BLOB_RESTORE_MANIFEST_RETENTION_MAX) {
 				return Void();
 			}
-			BlobManifest::deleteOldest(allFiles, writer);
+			wait(BlobManifest::deleteOldest(allFiles, writer));
```

Comment:
Are we interested in logging if delete operation fails for some reason? Also, do we want to handle the failures differently?

Reply:
current error handling is straightforward - if deletion fails, it throws the error and it'll try again next time when we generate manifest. Consider the deletion error should be very rare and transient.. current approach should be ok for now